plugins/hadoop-authorization-sentry-default/sentry-client/src/main/java/com/thinkbiganalytics/datalake/authorization/client/SentryClient.java

```diff
@@ -200,33 +206,62 @@ public boolean flushACL(String HadoopConfigurationResource, String allPathForAcl
 
     }
 
-
     /**
-     * Get default Kylo groups
+     * Get Kylo groups from Static List , Unix or LDAP
      */
     public List<HadoopAuthorizationGroup> getAllGroups() {
         List<HadoopAuthorizationGroup> sentryHadoopAuthorizationGroups = new ArrayList<>();
         SentryGroup hadoopAuthorizationGroup = new SentryGroup();
 
-        String sentryGroups = this.clientConfig.getSentryGroups();
-        List<String> sentryGroupsList = new ArrayList<String>(Arrays.asList(sentryGroups.split(",")));
+        String authorizationSentryGroupFetchType = this.clientConfig.getAuthorizationGroupType();
+
+
+        if(authorizationSentryGroupFetchType.equalsIgnoreCase("static"))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "static" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)